src/scenes/Event/components/EventTxHistory/resultHistory.js

```diff
@@ -53,9 +74,9 @@ export default class EventResultHistory extends Component {
               {_.map(sortedOracles, (oracle, index) => (
                 <TableRow key={`result-${index}`} selected={index % 2 === 1}>
                   <TableCell padding="dense">{getLocalDateTimeString(oracle.endTime)}</TableCell>
-                  <TableCell padding="dense">{oracle.token === Token.Qtum ? 'Betting Round' : `Voting Round ${index}`}</TableCell>
+                  <TableCell padding="dense">{this.checkType(oracle, index)}</TableCell>
                   <TableCell padding="dense">
-                    {index !== sortedOracles.length - 1
+                    {index !== sortedOracles.length - 1 && index !== 0
```

Comment:
maybe `> 0`?

Reply:
it's the same thing. since the index always >0